advocacy_docs/edb-postgres-ai/migration-etl/index.mdx

```diff
@@ -9,34 +9,71 @@ navigation:
   - migration-and-ai
 ---
 
-Moving your data to Postgres is a challenge that EDB Postgres AI is built to solve. The EDB Postgres AI platform includes a set of tools that help you migrate your data to Postgres and keep it up-to-date. 
+Moving your data to Postgres is a challenge that EDB Postgres AI is built to solve. The EDB Postgres AI platform includes a set of tools that help you migrate your data to Postgres and keep it up-to-date.
 
-These tools include the EDB Data Migration Service, EDB Postgres AI Migration Portal, Replication Server and Migration Toolkit.
+The available migration tools are suitable for different migration scenarios. The tool you use depends on the source database, the target database and the migration scenario.
 
-## The EDB Data Migration Service
+## EDB Data Migration Service
 
-The [EDB Data Migration Service](data-migration-service) is a cloud-based service that actively helps you migrate your data to Postgres.
+[EDB Data Migration Service](data-migration-service) is a cloud-based service that helps you migrate your data to an [EDB Hosted Cloud Service](/edb-postgres-ai/cloud-service/getting_started/planning/choosing_your_deployment/edb_hosted_cloud_service/) database.
 
-## The EDB Postgres AI Migration Portal
+EDB DMS offers a secure and fault-tolerant way to migrate database data to the EDB Postgres AI platform using change data capture and event streaming technologies. This allows the migration to stream any changes in the source database to the EDB Cloud Service database until you decide to cut over.
 
-The [EDB Postgres AI Migration Portal](/migration_portal/latest/) is a web-based tool that helps you plan and execute your migration to Postgres.
+At the end of an EDB DMS migration, you will host your database entirely within the EDB Postgres AI Cloud Service ecosystem.
```

Comment:
I'm struggling with this sentence; in particular, the phrase "you will host your database entirely within ...".  At the end of a data migration the data in your source database will have been copied to your EDB Postgres AI Cloud Service managed Postgres database.  Other migration steps are needed to completely have your application database entirely hosted in EDB's cloud service.